pyiron_atomistics/project.py

```diff
@@ -511,9 +511,16 @@ def _calculation_validation(
     def inspect_periodic_table():
         return PeriodicTable()
 
+ @staticmethod
+    def inspect_empirical_potentials():
+        return LammpsPotentialFile()
+
     @staticmethod
     def inspect_emperical_potentials():
```

Comment:
Between the `@staticmethod` and definition, we want `@deprecate("Use inspect_empirical_potentials instead!")`, with the corresponding `from pyiron_base.utils.deprecate import deprecate` at the head of the file.

@pmrv has some nice docstrings [in the deprecator](https://github.com/pyiron/pyiron_base/blob/5162288f402b956e73f43445e1231734d9c9dc48/pyiron_base/utils/deprecate.py#L26) so I think I'm suggesting the correct syntax, but try it out locally to make sure.